### DISCUSSION TITLE
New uboot-rockship Version

### DISCUSSION THREAD

**Discussion by yannick2801** [Ysurac/openmptcprouter#3597] [Q&A]
is it possible to migrated from 2023.04 version to the newer version 2024.04 ?

i am trying to compile a new image for Rock pi 4C+ but i need the lastest uboot version

Br
Yannick

**Comment by Ysurac** [comment]
I will update to 2024.10 but I would like to release version 0.61 before any update. I hope really soon.

**Comment by yannick2801** [comment]
will i be able to compile in openmptcprouter 5.4 version ?

BR

**Reply by Ysurac** [reply]
no, OpenMPTCProuter 5.4 is legacy. No more updates.

**Reply by yannick2801** [reply]
what can i do if i need the new uboot on a 5.4 version ? it's not possible ? 

Thanks

**Reply by Ysurac** [reply]
No idea, you can try to update the package but not sure if this will work or not.

**Reply by yannick2801** [reply]
ok Thanks, i will have a look.

For you, what version will be the best to compile for the ROCK pi 4 ?

**Reply by Ysurac** [reply]
The only current supported kernel on snapshot is 6.6

**Reply by yannick2801** [reply]
Thanks for your support. 
BR Yannick
